src/style.css

```diff
@@ -1,3 +1,129 @@
+html {
+  font-family: "Roboto", sans-serif;
+  --button-color: #00acdc;
+  --main-color: #060b35;
+  --secondary-color: #616070;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
```

Comment:
box-sizing: border-box is duplicated, please move it to the `*{ box-sizing: border-box; }`